vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthLoginFragment.kt

```diff
@@ -119,40 +120,43 @@ class FtueAuthLoginFragment @Inject constructor() : AbstractSSOFtueAuthFragment<
     }
 
     private fun submit() {
-        cleanupUi()
+        withState(viewModel) { state ->
```

Comment:
this block is an indentation change due to needing the `state` in order to create the `AuthenticateAction` from the `SignMode`, this isn't needed by the new combined fragments as they don't share the same fragment